lib/mimemagic/tables.rb

```diff
@@ -1,2154 +1,159 @@
 # -*- coding: binary -*-
 # frozen_string_literal: true
-# Generated from freedesktop.org.xml
+# Generated from script/freedesktop.org.xml
+require 'nokogiri'
```

Comment:
Need to add `s.add_dependency('nokogiri', '~> 1.11.2')`

Reply:
Thanks